datameta/api/apikeys.py

```diff
@@ -55,11 +57,11 @@ def __json__(self, request: Request) -> List[dict]:
         return self.apikeys
 
 
-def get_expiration_date_from_str(expires_str:Optional[str]):
+def get_expiration_date_from_str(expires_str: Optional[str]):
     """Validates a user-defined ApiKey expiration date and converts it into a datetime object"""
     # calculated the latest expiration date allowed
     max_expiration_period = int(threadlocal.get_current_registry().settings['datameta.apikeys.max_expiration_period'])
-    max_expiration_datetime = datetime.now() + timedelta(days=max_expiration_period)
+    max_expiration_datetime = datetime.now() + timedelta(days = max_expiration_period)
```

Comment:
that is a weird one, I thought the convention is without padding for function params

Reply:
Indeed, I think that one can in accidentally and is not captured due to the flake8 ignore list. I'll revert it.